src/components/global/Button.jsx

```diff
@@ -1,16 +1,44 @@
 import React from "react";
 import { Link } from "react-router-dom";
+
```

Comment:
Did you run the linter?

Reply:
Yep I did, there was one warning that I wasn't sure about, but I've just fixed it